app/helpers/likes_helper.rb

```diff
@@ -0,0 +1,2 @@
+module LikeHelper
+end
```

Comment:
- [ ] Let's correct this error
![Screenshot from 2022-02-26 12-54-22](https://user-images.githubusercontent.com/65860678/155840594-72b73936-e513-4a61-8bd0-f3d43869fa20.png)